ansible/modules/hashivault/hashivault_list.py

```diff
@@ -76,7 +76,7 @@ def hashivault_list(params):
     client = hashivault_auth_client(params)
     version = params.get('version')
     mount_point = params.get('mount_point')
-    secret = params.get('secret')
+    secret = params.get('secret').lstrip()
```

Comment:
I am not sure you wanted to keep this to remove spaces at the beginning.

Reply:
Yeh, that doesn't make a lot of sense